src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationContext.cs

```diff
@@ -237,11 +238,28 @@ public async Task<DeviceCodeResult> AcquireDeviceCodeAsync(string resource, stri
 #if iOS
         [Obsolete("This device profile API should only be used on text-only devices, and not on this target platform which is offering an interactive authentication experience. For details please see https://aka.ms/AdalNetConfFlows")]
 #endif
-        public async Task<AuthenticationResult> AcquireTokenByDeviceCodeAsync(DeviceCodeResult deviceCodeResult)
+        public Task<AuthenticationResult> AcquireTokenByDeviceCodeAsync(DeviceCodeResult deviceCodeResult)
```

Comment:
async ?

Reply:
This doesn't need to be async since we're just returning the task from the other method.  There's no need to do an await here since we're just calling another method that returns a task.

Reply:
Will it break anyone by removing it?

Reply:
No.  the async keyword isn't actually part of the public API per se, it's a compiler hint for how to thread the call.  The important part is that it still returns Task<AuthenticationResult>